models/articles.model.js

```diff
@@ -91,3 +91,18 @@ exports.insertComment = (newComment) => {
       return rows[0];
     });
 };
+
+exports.updateArticle = (article_id, inc_votes) =>
+{
+    return db.query(
+            `UPDATE articles
+                SET votes = votes + $1
+                WHERE article_id = $2
+                RETURNING *`,
+            [inc_votes, article_id]
+        )
+        .then(({ rows }) =>
```

Comment:
We also need to add a custom 404 error
Status 404, non existent ID, e.g. 0 or 9999